frontend/src/libs/components/modal/steps/delete-account/delete-account-confirmation.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button } from '#libs/components/components.js';
```

Comment:
should be in pages/profile-settings/components/delete-account-modal/delete-account-modal.tsx 

1. we need to export only one component
2. all these logic with steps should be moved there, it's a common modal component, it shouldn't know about other features

Reply:
hi, @Liza-Veis Please check these changes, I will try to fix the other comments as soon as I can.